Adding additional cartridges and file parser

ifilot opened the pull request:
* Adding additional cartridges to the collection
  - Basic 1.0 in English
  - Text 1 (German edition)
  - Text 2 (German edition) 
  - "Bildschirmtext" (German)
  - Maintenance (modified version)
* A parser script (`readme-generator.py`) that parses the `cartridges.json` file to produce nicely formatted output, checks for potential duplicates and also shows the MD5 checksum in the README file.

dionoid commented:
Hi Ivo, thanks for the PR. A few remarks:

- The updated README.md now shows a floppy disk icon on each line, which makes me think the cartridge roms are actually floppy disk dumps. What is the reason for this?
- Except for the cartridge "BASICEN1.0.bin", you didn't specify any descriptions for the cartridges you added.  E.g. what do the cartridges "Text 1" and "Text 2" contain? I don't have a clue :)
- I like how the readme-generator.py script checks if all roms in cartridges.json actually exist as file. It would be nice if the script could also check/warn for .bin rom files that are not represented in cartridges.json. Is it doable to add this?

Thanks, Dion

ifilot commented:
Thank you for the review and the feedback.

To answer your remarks on a point-by-point basis:

* It was nothing more than some eye-candy, but I agree with you, it is misleading. We could potentially differentiate between a cartridge and a floppy disk, but the [icon set](https://gist.github.com/rxaviers/7360908) of Github does not have a nice icon for a cartridge. I have removed the icon, but I am happy to reintroduce it if we can come up with some use for it.
* Text1 and Text2 are essentially the German versions of Tekst 1. In other words, they are version 1 and 2 of a word processor. I have expanded upon the description.
* This is certainly doable and I have added this functionality. With the introduction of this functionality I actually noted that there was no entry for `Tekst 1.bin`. Thank you for this excellent suggestion.

Let me know if these revisions are suitable. Always happy to make another iteration.

dionoid commented:
That was fast! :) Excellent changes.
